apps/fabric-website/src/components/App/App.tsx

```diff
@@ -61,7 +61,7 @@ export class App extends React.Component<IAppProps, any> {
     if (nextProps && nextProps.children !== this.props.children) {
       const hash = extractAnchorLink(window.location.hash);
       // when loading a new component page, we want to scroll to the top
-      if (hash.indexOf('/components') !== -1) {
+      if (hash && hash.indexOf('/components') !== -1) {
```

Comment:
If `extractAnchrorLink` is not returning the right type and can't be fixed, may be better to cast return type as `whatever | undefined`?

Reply:
This got me thinking more and I realized the bug is different than I originally thought--the issue is actually that it wasn't correctly handling partial URLs like `#/components/checkbox`. So now I've fixed that actual issue.